tests/utils.py

```diff
@@ -149,3 +151,25 @@ def with_dataset(client, name=None, commit_database=False):
     if commit_database:
         client.get_database().register(dataset)
         client.get_database().commit()
+
+
+def retry_failed(fn=None, extended: bool = False):
```

Comment:
Should we replace `extended` with `max_runs=5`? This would allow us to have more than two possible values if needed in future.

Reply:
The original motivation for switching to this fixture was the possibility to handle the number of repetitions in a single place (e.g. switching to 1 rep when testing locally).
I would start with 2 categories (for standard vs occasionally failing tests) without the possibility to set a specific number of repetitions, and make this more flexible later if we need it.

A simpler solution would be using the original decorator and create constants to pass to `flaky(max_runs)`